<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head><title>R: Tools for Single Cell Genomics</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="R.css" />
</head><body>
<h1> Tools for Single Cell Genomics
<img class="toplogo" src="../../../doc/html/Rlogo.svg" alt="[R logo]" />
</h1>
<hr/>
<div style="text-align: center;">
<a href="../../../doc/html/packages.html"><img class="arrow" src="../../../doc/html/left.jpg" alt="[Up]" /></a>
<a href="../../../doc/html/index.html"><img class="arrow" src="../../../doc/html/up.jpg" alt="[Top]" /></a>
</div><h2>Documentation for package &lsquo;Seurat&rsquo; version 3.0.0.9000</h2>

<ul><li><a href="../DESCRIPTION">DESCRIPTION file</a>.</li>
</ul>

<h2>Help Pages</h2>


<p style="text-align: center;">
<a href="#A">A</a>
<a href="#B">B</a>
<a href="#C">C</a>
<a href="#D">D</a>
<a href="#E">E</a>
<a href="#F">F</a>
<a href="#G">G</a>
<a href="#H">H</a>
<a href="#I">I</a>
<a href="#J">J</a>
<a href="#K">K</a>
<a href="#L">L</a>
<a href="#M">M</a>
<a href="#N">N</a>
<a href="#O">O</a>
<a href="#P">P</a>
<a href="#R">R</a>
<a href="#S">S</a>
<a href="#T">T</a>
<a href="#U">U</a>
<a href="#V">V</a>
<a href="#W">W</a>
<a href="#misc">misc</a>
</p>


<h2><a name="A">-- A --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="AddMetaData.html">AddMetaData</a></td>
<td>Add in metadata associated with either cells or features.</td></tr>
<tr><td style="width: 25%;"><a href="AddMetaData.html">AddMetaData.Assay</a></td>
<td>Add in metadata associated with either cells or features.</td></tr>
<tr><td style="width: 25%;"><a href="AddMetaData.html">AddMetaData.Seurat</a></td>
<td>Add in metadata associated with either cells or features.</td></tr>
<tr><td style="width: 25%;"><a href="AddModuleScore.html">AddModuleScore</a></td>
<td>Calculate module scores for featre expression programs in single cells</td></tr>
<tr><td style="width: 25%;"><a href="ALRAChooseKPlot.html">ALRAChooseKPlot</a></td>
<td>ALRA Approximate Rank Selection Plot</td></tr>
<tr><td style="width: 25%;"><a href="AnchorSet-class.html">AnchorSet</a></td>
<td>The AnchorSet Class</td></tr>
<tr><td style="width: 25%;"><a href="AnchorSet-class.html">AnchorSet-class</a></td>
<td>The AnchorSet Class</td></tr>
<tr><td style="width: 25%;"><a href="as.sparse.html">as.data.frame.Matrix</a></td>
<td>Convert between data frames and sparse matrices</td></tr>
<tr><td style="width: 25%;"><a href="Convert.html">as.Seurat</a></td>
<td>Convert Seurat objects to other classes and vice versa</td></tr>
<tr><td style="width: 25%;"><a href="Convert.html">as.Seurat.SingleCellExperiment</a></td>
<td>Convert Seurat objects to other classes and vice versa</td></tr>
<tr><td style="width: 25%;"><a href="Convert.html">as.SingleCellExperiment</a></td>
<td>Convert Seurat objects to other classes and vice versa</td></tr>
<tr><td style="width: 25%;"><a href="Convert.html">as.SingleCellExperiment.Seurat</a></td>
<td>Convert Seurat objects to other classes and vice versa</td></tr>
<tr><td style="width: 25%;"><a href="as.sparse.html">as.sparse</a></td>
<td>Convert between data frames and sparse matrices</td></tr>
<tr><td style="width: 25%;"><a href="as.sparse.html">as.sparse.data.frame</a></td>
<td>Convert between data frames and sparse matrices</td></tr>
<tr><td style="width: 25%;"><a href="Assay-class.html">Assay</a></td>
<td>The Assay Class</td></tr>
<tr><td style="width: 25%;"><a href="Assay-class.html">Assay-class</a></td>
<td>The Assay Class</td></tr>
<tr><td style="width: 25%;"><a href="AugmentPlot.html">AugmentPlot</a></td>
<td>Augments ggplot2-based plot with a PNG image.</td></tr>
<tr><td style="width: 25%;"><a href="AverageExpression.html">AverageExpression</a></td>
<td>Averaged feature expression by identity class</td></tr>
</table>

<h2><a name="B">-- B --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="BarcodeInflectionsPlot.html">BarcodeInflectionsPlot</a></td>
<td>Plot the Barcode Distribution and Calculated Inflection Points</td></tr>
<tr><td style="width: 25%;"><a href="CustomPalette.html">BlackAndWhite</a></td>
<td>Create a custom color palette</td></tr>
<tr><td style="width: 25%;"><a href="CustomPalette.html">BlueAndRed</a></td>
<td>Create a custom color palette</td></tr>
<tr><td style="width: 25%;"><a href="SeuratTheme.html">BoldTitle</a></td>
<td>Seurat Themes</td></tr>
<tr><td style="width: 25%;"><a href="BuildClusterTree.html">BuildClusterTree</a></td>
<td>Phylogenetic Analysis of Identity Classes</td></tr>
</table>

<h2><a name="C">-- C --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="CalculateBarcodeInflections.html">CalculateBarcodeInflections</a></td>
<td>Calculate the Barcode Distribution Inflection</td></tr>
<tr><td style="width: 25%;"><a href="CaseMatch.html">CaseMatch</a></td>
<td>Match the case of character vectors</td></tr>
<tr><td style="width: 25%;"><a href="cc.genes.html">cc.genes</a></td>
<td>Cell cycle genes</td></tr>
<tr><td style="width: 25%;"><a href="CellCycleScoring.html">CellCycleScoring</a></td>
<td>Score cell cycle phases</td></tr>
<tr><td style="width: 25%;"><a href="CellScatter.html">CellPlot</a></td>
<td>Cell-cell scatter plot</td></tr>
<tr><td style="width: 25%;"><a href="Cells.html">Cells</a></td>
<td>Get cells present in an object</td></tr>
<tr><td style="width: 25%;"><a href="Cells.html">Cells.default</a></td>
<td>Get cells present in an object</td></tr>
<tr><td style="width: 25%;"><a href="CellScatter.html">CellScatter</a></td>
<td>Cell-cell scatter plot</td></tr>
<tr><td style="width: 25%;"><a href="CollapseSpeciesExpressionMatrix.html">CollapseSpeciesExpressionMatrix</a></td>
<td>Slim down a multi-species expression matrix, when only one species is primarily of interenst.</td></tr>
<tr><td style="width: 25%;"><a href="CombinePlots.html">CombinePlots</a></td>
<td>Combine ggplot2-based plots into a single plot</td></tr>
<tr><td style="width: 25%;"><a href="Command.html">Command</a></td>
<td>Get SeuratCommands</td></tr>
<tr><td style="width: 25%;"><a href="Command.html">Command.Seurat</a></td>
<td>Get SeuratCommands</td></tr>
<tr><td style="width: 25%;"><a href="Convert.html">Convert</a></td>
<td>Convert Seurat objects to other classes and vice versa</td></tr>
<tr><td style="width: 25%;"><a href="Convert.html">Convert.anndata.base.AnnData</a></td>
<td>Convert Seurat objects to other classes and vice versa</td></tr>
<tr><td style="width: 25%;"><a href="Convert.html">Convert.seurat</a></td>
<td>Convert Seurat objects to other classes and vice versa</td></tr>
<tr><td style="width: 25%;"><a href="CreateAssayObject.html">CreateAssayObject</a></td>
<td>Create an Assay object</td></tr>
<tr><td style="width: 25%;"><a href="CreateDimReducObject.html">CreateDimReducObject</a></td>
<td>Create a DimReduc object</td></tr>
<tr><td style="width: 25%;"><a href="CreateSeuratObject.html">CreateSeuratObject</a></td>
<td>Create a Seurat object</td></tr>
<tr><td style="width: 25%;"><a href="CustomDistance.html">CustomDistance</a></td>
<td>Run a custom distance function on an input data matrix</td></tr>
<tr><td style="width: 25%;"><a href="CustomPalette.html">CustomPalette</a></td>
<td>Create a custom color palette</td></tr>
</table>

<h2><a name="D">-- D --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="SeuratTheme.html">DarkTheme</a></td>
<td>Seurat Themes</td></tr>
<tr><td style="width: 25%;"><a href="DefaultAssay.html">DefaultAssay</a></td>
<td>Get the default assay</td></tr>
<tr><td style="width: 25%;"><a href="DefaultAssay.html">DefaultAssay.DimReduc</a></td>
<td>Get the default assay</td></tr>
<tr><td style="width: 25%;"><a href="DefaultAssay.html">DefaultAssay.Seurat</a></td>
<td>Get the default assay</td></tr>
<tr><td style="width: 25%;"><a href="DefaultAssay.html">DefaultAssay&lt;-</a></td>
<td>Get the default assay</td></tr>
<tr><td style="width: 25%;"><a href="DefaultAssay.html">DefaultAssay&lt;-.Seurat</a></td>
<td>Get the default assay</td></tr>
<tr><td style="width: 25%;"><a href="DietSeurat.html">DietSeurat</a></td>
<td>Slim down a Seurat object</td></tr>
<tr><td style="width: 25%;"><a href="DimHeatmap.html">DimHeatmap</a></td>
<td>Dimensional reduction heatmap</td></tr>
<tr><td style="width: 25%;"><a href="DimPlot.html">DimPlot</a></td>
<td>Dimensional reduction plot</td></tr>
<tr><td style="width: 25%;"><a href="DimReduc-class.html">DimReduc</a></td>
<td>The Dimmensional Reduction Class</td></tr>
<tr><td style="width: 25%;"><a href="DimReduc-class.html">DimReduc-class</a></td>
<td>The Dimmensional Reduction Class</td></tr>
<tr><td style="width: 25%;"><a href="DoHeatmap.html">DoHeatmap</a></td>
<td>Feature expression heatmap</td></tr>
<tr><td style="width: 25%;"><a href="DotPlot.html">DotPlot</a></td>
<td>Dot plot visualization</td></tr>
</table>

<h2><a name="E">-- E --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="ElbowPlot.html">ElbowPlot</a></td>
<td>Quickly Pick Relevant Dimensions</td></tr>
<tr><td style="width: 25%;"><a href="Embeddings.html">Embeddings</a></td>
<td>Get cell embeddings</td></tr>
<tr><td style="width: 25%;"><a href="Embeddings.html">Embeddings.DimReduc</a></td>
<td>Get cell embeddings</td></tr>
<tr><td style="width: 25%;"><a href="Embeddings.html">Embeddings.Seurat</a></td>
<td>Get cell embeddings</td></tr>
<tr><td style="width: 25%;"><a href="ExpMean.html">ExpMean</a></td>
<td>Calculate the mean of logged values</td></tr>
<tr><td style="width: 25%;"><a href="ExportToCellbrowser.html">ExportToCellbrowser</a></td>
<td>Export Seurat object for UCSC cell browser</td></tr>
<tr><td style="width: 25%;"><a href="ExpSD.html">ExpSD</a></td>
<td>Calculate the standard deviation of logged values</td></tr>
<tr><td style="width: 25%;"><a href="ExpVar.html">ExpVar</a></td>
<td>Calculate the variance of logged values</td></tr>
</table>

<h2><a name="F">-- F --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="FeaturePlot.html">FeatureHeatmap</a></td>
<td>Visualize 'features' on a dimensional reduction plot</td></tr>
<tr><td style="width: 25%;"><a href="FeatureLocator.html">FeatureLocator</a></td>
<td>Feature Locator</td></tr>
<tr><td style="width: 25%;"><a href="FeaturePlot.html">FeaturePlot</a></td>
<td>Visualize 'features' on a dimensional reduction plot</td></tr>
<tr><td style="width: 25%;"><a href="FeatureScatter.html">FeatureScatter</a></td>
<td>Scatter plot of single cell data</td></tr>
<tr><td style="width: 25%;"><a href="FetchData.html">FetchData</a></td>
<td>Access cellular data</td></tr>
<tr><td style="width: 25%;"><a href="FindAllMarkers.html">FindAllMarkers</a></td>
<td>Gene expression markers for all identity classes</td></tr>
<tr><td style="width: 25%;"><a href="FindAllMarkers.html">FindAllMarkersNode</a></td>
<td>Gene expression markers for all identity classes</td></tr>
<tr><td style="width: 25%;"><a href="FindClusters.html">FindClusters</a></td>
<td>Cluster Determination</td></tr>
<tr><td style="width: 25%;"><a href="FindClusters.html">FindClusters.default</a></td>
<td>Cluster Determination</td></tr>
<tr><td style="width: 25%;"><a href="FindClusters.html">FindClusters.Seurat</a></td>
<td>Cluster Determination</td></tr>
<tr><td style="width: 25%;"><a href="FindConservedMarkers.html">FindConservedMarkers</a></td>
<td>Finds markers that are conserved between the two groups</td></tr>
<tr><td style="width: 25%;"><a href="FindIntegrationAnchors.html">FindIntegrationAnchors</a></td>
<td>Find integration anchors</td></tr>
<tr><td style="width: 25%;"><a href="FindMarkers.html">FindMarkers</a></td>
<td>Gene expression markers of identity classes</td></tr>
<tr><td style="width: 25%;"><a href="FindMarkers.html">FindMarkers.default</a></td>
<td>Gene expression markers of identity classes</td></tr>
<tr><td style="width: 25%;"><a href="FindMarkers.html">FindMarkers.Seurat</a></td>
<td>Gene expression markers of identity classes</td></tr>
<tr><td style="width: 25%;"><a href="FindMarkers.html">FindMarkersNode</a></td>
<td>Gene expression markers of identity classes</td></tr>
<tr><td style="width: 25%;"><a href="FindNeighbors.html">FindNeighbors</a></td>
<td>SNN Graph Construction</td></tr>
<tr><td style="width: 25%;"><a href="FindNeighbors.html">FindNeighbors.Assay</a></td>
<td>SNN Graph Construction</td></tr>
<tr><td style="width: 25%;"><a href="FindNeighbors.html">FindNeighbors.default</a></td>
<td>SNN Graph Construction</td></tr>
<tr><td style="width: 25%;"><a href="FindNeighbors.html">FindNeighbors.dist</a></td>
<td>SNN Graph Construction</td></tr>
<tr><td style="width: 25%;"><a href="FindNeighbors.html">FindNeighbors.Seurat</a></td>
<td>SNN Graph Construction</td></tr>
<tr><td style="width: 25%;"><a href="FindTransferAnchors.html">FindTransferAnchors</a></td>
<td>Find transfer anchors</td></tr>
<tr><td style="width: 25%;"><a href="FindVariableFeatures.html">FindVariableFeatures</a></td>
<td>Find variable features</td></tr>
<tr><td style="width: 25%;"><a href="FindVariableFeatures.html">FindVariableFeatures.Assay</a></td>
<td>Find variable features</td></tr>
<tr><td style="width: 25%;"><a href="FindVariableFeatures.html">FindVariableFeatures.default</a></td>
<td>Find variable features</td></tr>
<tr><td style="width: 25%;"><a href="FindVariableFeatures.html">FindVariableFeatures.Seurat</a></td>
<td>Find variable features</td></tr>
<tr><td style="width: 25%;"><a href="FindVariableFeatures.html">FindVariableGenes</a></td>
<td>Find variable features</td></tr>
<tr><td style="width: 25%;"><a href="SeuratTheme.html">FontSize</a></td>
<td>Seurat Themes</td></tr>
</table>

<h2><a name="G">-- G --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="FeatureScatter.html">GenePlot</a></td>
<td>Scatter plot of single cell data</td></tr>
<tr><td style="width: 25%;"><a href="GetAssay.html">GetAssay</a></td>
<td>Get an assay from an object</td></tr>
<tr><td style="width: 25%;"><a href="GetAssay.html">GetAssay.Seurat</a></td>
<td>Get an assay from an object</td></tr>
<tr><td style="width: 25%;"><a href="GetAssayData.html">GetAssayData</a></td>
<td>Accessor function for multimodal data</td></tr>
<tr><td style="width: 25%;"><a href="GetAssayData.html">GetAssayData.Assay</a></td>
<td>Accessor function for multimodal data</td></tr>
<tr><td style="width: 25%;"><a href="GetAssayData.html">GetAssayData.Seurat</a></td>
<td>Accessor function for multimodal data</td></tr>
<tr><td style="width: 25%;"><a href="GetIntegrationData.html">GetIntegrationData</a></td>
<td>Get integation data</td></tr>
<tr><td style="width: 25%;"><a href="Graph-class.html">Graph</a></td>
<td>The Graph Class</td></tr>
<tr><td style="width: 25%;"><a href="Graph-class.html">Graph-class</a></td>
<td>The Graph Class</td></tr>
</table>

<h2><a name="H">-- H --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="HoverLocator.html">HoverLocator</a></td>
<td>Hover Locator</td></tr>
<tr><td style="width: 25%;"><a href="HTODemux.html">HTODemux</a></td>
<td>Demultiplex samples based on data from cell 'hashing'</td></tr>
<tr><td style="width: 25%;"><a href="HTOHeatmap.html">HTOHeatmap</a></td>
<td>Hashtag oligo heatmap</td></tr>
<tr><td style="width: 25%;"><a href="HVFInfo.html">HVFInfo</a></td>
<td>Get highly variable feature information</td></tr>
<tr><td style="width: 25%;"><a href="HVFInfo.html">HVFInfo.Assay</a></td>
<td>Get highly variable feature information</td></tr>
<tr><td style="width: 25%;"><a href="HVFInfo.html">HVFInfo.Seurat</a></td>
<td>Get highly variable feature information</td></tr>
</table>

<h2><a name="I">-- I --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="DimPlot.html">ICAPlot</a></td>
<td>Dimensional reduction plot</td></tr>
<tr><td style="width: 25%;"><a href="Idents.html">Idents</a></td>
<td>Get, set, and manipulate an object's identity classes</td></tr>
<tr><td style="width: 25%;"><a href="Idents.html">Idents.Seurat</a></td>
<td>Get, set, and manipulate an object's identity classes</td></tr>
<tr><td style="width: 25%;"><a href="Idents.html">Idents&lt;-</a></td>
<td>Get, set, and manipulate an object's identity classes</td></tr>
<tr><td style="width: 25%;"><a href="Idents.html">Idents&lt;-.Seurat</a></td>
<td>Get, set, and manipulate an object's identity classes</td></tr>
<tr><td style="width: 25%;"><a href="IntegrateData.html">IntegrateData</a></td>
<td>Integrate data</td></tr>
<tr><td style="width: 25%;"><a href="IntegrationData-class.html">IntegrationData</a></td>
<td>The IntegrationData Class</td></tr>
<tr><td style="width: 25%;"><a href="IntegrationData-class.html">IntegrationData-class</a></td>
<td>The IntegrationData Class</td></tr>
</table>

<h2><a name="J">-- J --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="JackStraw.html">JackStraw</a></td>
<td>Determine statistical significance of PCA scores.</td></tr>
<tr><td style="width: 25%;"><a href="JackStrawData-class.html">JackStrawData</a></td>
<td>The JackStrawData Class</td></tr>
<tr><td style="width: 25%;"><a href="JackStrawData-class.html">JackStrawData-class</a></td>
<td>The JackStrawData Class</td></tr>
<tr><td style="width: 25%;"><a href="JackStrawPlot.html">JackStrawPlot</a></td>
<td>JackStraw Plot</td></tr>
<tr><td style="width: 25%;"><a href="JS.html">JS</a></td>
<td>Get JackStraw information</td></tr>
<tr><td style="width: 25%;"><a href="JS.html">JS.DimReduc</a></td>
<td>Get JackStraw information</td></tr>
<tr><td style="width: 25%;"><a href="JS.html">JS.JackStrawData</a></td>
<td>Get JackStraw information</td></tr>
<tr><td style="width: 25%;"><a href="JS.html">JS&lt;-</a></td>
<td>Get JackStraw information</td></tr>
<tr><td style="width: 25%;"><a href="JS.html">JS&lt;-.DimReduc</a></td>
<td>Get JackStraw information</td></tr>
<tr><td style="width: 25%;"><a href="JS.html">JS&lt;-.JackStrawData</a></td>
<td>Get JackStraw information</td></tr>
</table>

<h2><a name="K">-- K --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="Key.html">Key</a></td>
<td>Get a key</td></tr>
<tr><td style="width: 25%;"><a href="Key.html">Key.Assay</a></td>
<td>Get a key</td></tr>
<tr><td style="width: 25%;"><a href="Key.html">Key.DimReduc</a></td>
<td>Get a key</td></tr>
<tr><td style="width: 25%;"><a href="Key.html">Key.Seurat</a></td>
<td>Get a key</td></tr>
<tr><td style="width: 25%;"><a href="Key.html">Key&lt;-</a></td>
<td>Get a key</td></tr>
<tr><td style="width: 25%;"><a href="Key.html">Key&lt;-.Assay</a></td>
<td>Get a key</td></tr>
<tr><td style="width: 25%;"><a href="Key.html">Key&lt;-.DimReduc</a></td>
<td>Get a key</td></tr>
</table>

<h2><a name="L">-- L --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="L2CCA.html">L2CCA</a></td>
<td>L2-Normalize CCA</td></tr>
<tr><td style="width: 25%;"><a href="L2Dim.html">L2Dim</a></td>
<td>L2-normalization</td></tr>
<tr><td style="width: 25%;"><a href="LabelClusters.html">LabelClusters</a></td>
<td>Label clusters on a ggplot2-based scatter plot</td></tr>
<tr><td style="width: 25%;"><a href="LabelPoints.html">Labeler</a></td>
<td>Add text labels to a ggplot2 plot</td></tr>
<tr><td style="width: 25%;"><a href="LabelPoints.html">LabelPoints</a></td>
<td>Add text labels to a ggplot2 plot</td></tr>
<tr><td style="width: 25%;"><a href="Idents.html">levels.Seurat</a></td>
<td>Get, set, and manipulate an object's identity classes</td></tr>
<tr><td style="width: 25%;"><a href="Idents.html">levels&lt;-.Seurat</a></td>
<td>Get, set, and manipulate an object's identity classes</td></tr>
<tr><td style="width: 25%;"><a href="Loadings.html">Loadings</a></td>
<td>Get feature loadings</td></tr>
<tr><td style="width: 25%;"><a href="Loadings.html">Loadings.DimReduc</a></td>
<td>Get feature loadings</td></tr>
<tr><td style="width: 25%;"><a href="Loadings.html">Loadings.Seurat</a></td>
<td>Get feature loadings</td></tr>
<tr><td style="width: 25%;"><a href="Loadings.html">Loadings&lt;-</a></td>
<td>Get feature loadings</td></tr>
<tr><td style="width: 25%;"><a href="Loadings.html">Loadings&lt;-.DimReduc</a></td>
<td>Get feature loadings</td></tr>
<tr><td style="width: 25%;"><a href="LocalStruct.html">LocalStruct</a></td>
<td>Calculate the local structure preservation metric</td></tr>
<tr><td style="width: 25%;"><a href="LogNormalize.html">LogNormalize</a></td>
<td>Normalize raw data</td></tr>
<tr><td style="width: 25%;"><a href="LogVMR.html">LogVMR</a></td>
<td>Calculate the variance to mean ratio of logged values</td></tr>
</table>

<h2><a name="M">-- M --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="VariableFeaturePlot.html">MeanVarPlot</a></td>
<td>View variable features</td></tr>
<tr><td style="width: 25%;"><a href="merge.Seurat.html">merge</a></td>
<td>Merge Seurat Objects</td></tr>
<tr><td style="width: 25%;"><a href="merge.Seurat.html">merge.Assay</a></td>
<td>Merge Seurat Objects</td></tr>
<tr><td style="width: 25%;"><a href="merge.Seurat.html">merge.Seurat</a></td>
<td>Merge Seurat Objects</td></tr>
<tr><td style="width: 25%;"><a href="MetaFeature.html">MetaFeature</a></td>
<td>Aggregate expression of multiple features into a single feature</td></tr>
<tr><td style="width: 25%;"><a href="MinMax.html">MinMax</a></td>
<td>Apply a ceiling and floor to all values in a matrix</td></tr>
<tr><td style="width: 25%;"><a href="Misc.html">Misc</a></td>
<td>Access miscellaneous data</td></tr>
<tr><td style="width: 25%;"><a href="Misc.html">Misc.Seurat</a></td>
<td>Access miscellaneous data</td></tr>
<tr><td style="width: 25%;"><a href="Misc.html">Misc&lt;-</a></td>
<td>Access miscellaneous data</td></tr>
<tr><td style="width: 25%;"><a href="Misc.html">Misc&lt;-.Seurat</a></td>
<td>Access miscellaneous data</td></tr>
<tr><td style="width: 25%;"><a href="MixingMetric.html">MixingMetric</a></td>
<td>Calculates a mixing metric</td></tr>
</table>

<h2><a name="N">-- N --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="SeuratTheme.html">NoAxes</a></td>
<td>Seurat Themes</td></tr>
<tr><td style="width: 25%;"><a href="SeuratTheme.html">NoGrid</a></td>
<td>Seurat Themes</td></tr>
<tr><td style="width: 25%;"><a href="SeuratTheme.html">NoLegend</a></td>
<td>Seurat Themes</td></tr>
<tr><td style="width: 25%;"><a href="NormalizeData.html">NormalizeData</a></td>
<td>Normalize Data</td></tr>
<tr><td style="width: 25%;"><a href="NormalizeData.html">NormalizeData.Assay</a></td>
<td>Normalize Data</td></tr>
<tr><td style="width: 25%;"><a href="NormalizeData.html">NormalizeData.default</a></td>
<td>Normalize Data</td></tr>
<tr><td style="width: 25%;"><a href="NormalizeData.html">NormalizeData.Seurat</a></td>
<td>Normalize Data</td></tr>
</table>

<h2><a name="O">-- O --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="OldWhichCells.html">OldWhichCells</a></td>
<td>Identify cells matching certain criteria</td></tr>
<tr><td style="width: 25%;"><a href="OldWhichCells.html">OldWhichCells.Assay</a></td>
<td>Identify cells matching certain criteria</td></tr>
<tr><td style="width: 25%;"><a href="OldWhichCells.html">OldWhichCells.Seurat</a></td>
<td>Identify cells matching certain criteria</td></tr>
</table>

<h2><a name="P">-- P --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="pbmc_small.html">pbmc_small</a></td>
<td>A small example version of the PBMC dataset</td></tr>
<tr><td style="width: 25%;"><a href="DimPlot.html">PCAPlot</a></td>
<td>Dimensional reduction plot</td></tr>
<tr><td style="width: 25%;"><a href="PCASigGenes.html">PCASigGenes</a></td>
<td>Significant genes from a PCA</td></tr>
<tr><td style="width: 25%;"><a href="PolyDimPlot.html">PolyDimPlot</a></td>
<td>Polygon DimPlot</td></tr>
<tr><td style="width: 25%;"><a href="PolyFeaturePlot.html">PolyFeaturePlot</a></td>
<td>Polygon FeaturePlot</td></tr>
<tr><td style="width: 25%;"><a href="print.DimReduc.html">print</a></td>
<td>Print the results of a dimensional reduction analysis</td></tr>
<tr><td style="width: 25%;"><a href="print.DimReduc.html">print.DimReduc</a></td>
<td>Print the results of a dimensional reduction analysis</td></tr>
<tr><td style="width: 25%;"><a href="ProjectDim.html">ProjectDim</a></td>
<td>Project Dimensional reduction onto full dataset</td></tr>
<tr><td style="width: 25%;"><a href="CustomPalette.html">PurpleAndYellow</a></td>
<td>Create a custom color palette</td></tr>
</table>

<h2><a name="R">-- R --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="Read10X.html">Read10X</a></td>
<td>Load in data from 10X</td></tr>
<tr><td style="width: 25%;"><a href="Read10X_h5.html">Read10X_h5</a></td>
<td>Read 10X hdf5 file</td></tr>
<tr><td style="width: 25%;"><a href="ReadAlevin.html">ReadAlevin</a></td>
<td>Load in data from Alevin pipeline</td></tr>
<tr><td style="width: 25%;"><a href="ReadAlevinCsv.html">ReadAlevinCsv</a></td>
<td>Load in data from Alevin pipeline</td></tr>
<tr><td style="width: 25%;"><a href="RegressRegNB.html">RegressRegNB</a></td>
<td>Use regularized negative binomial regression to normalize UMI count data</td></tr>
<tr><td style="width: 25%;"><a href="RelativeCounts.html">RelativeCounts</a></td>
<td>Normalize raw data to fractions</td></tr>
<tr><td style="width: 25%;"><a href="RenameCells.html">RenameCells</a></td>
<td>Rename cells</td></tr>
<tr><td style="width: 25%;"><a href="RenameCells.html">RenameCells.Assay</a></td>
<td>Rename cells</td></tr>
<tr><td style="width: 25%;"><a href="RenameCells.html">RenameCells.DimReduc</a></td>
<td>Rename cells</td></tr>
<tr><td style="width: 25%;"><a href="RenameCells.html">RenameCells.Seurat</a></td>
<td>Rename cells</td></tr>
<tr><td style="width: 25%;"><a href="Idents.html">RenameIdent</a></td>
<td>Get, set, and manipulate an object's identity classes</td></tr>
<tr><td style="width: 25%;"><a href="Idents.html">RenameIdents</a></td>
<td>Get, set, and manipulate an object's identity classes</td></tr>
<tr><td style="width: 25%;"><a href="Idents.html">RenameIdents.Seurat</a></td>
<td>Get, set, and manipulate an object's identity classes</td></tr>
<tr><td style="width: 25%;"><a href="Idents.html">ReorderIdent</a></td>
<td>Get, set, and manipulate an object's identity classes</td></tr>
<tr><td style="width: 25%;"><a href="Idents.html">ReorderIdent.Seurat</a></td>
<td>Get, set, and manipulate an object's identity classes</td></tr>
<tr><td style="width: 25%;"><a href="SeuratTheme.html">RestoreLegend</a></td>
<td>Seurat Themes</td></tr>
<tr><td style="width: 25%;"><a href="RidgePlot.html">RidgePlot</a></td>
<td>Single cell ridge plot</td></tr>
<tr><td style="width: 25%;"><a href="SeuratTheme.html">RotatedAxis</a></td>
<td>Seurat Themes</td></tr>
<tr><td style="width: 25%;"><a href="RunALRA.html">RunALRA</a></td>
<td>Run Adaptively-thresholded Low Rank Approximation (ALRA)</td></tr>
<tr><td style="width: 25%;"><a href="RunALRA.html">RunALRA.default</a></td>
<td>Run Adaptively-thresholded Low Rank Approximation (ALRA)</td></tr>
<tr><td style="width: 25%;"><a href="RunALRA.html">RunALRA.Seurat</a></td>
<td>Run Adaptively-thresholded Low Rank Approximation (ALRA)</td></tr>
<tr><td style="width: 25%;"><a href="RunCCA.html">RunCCA</a></td>
<td>Perform Canonical Correlation Analysis</td></tr>
<tr><td style="width: 25%;"><a href="RunCCA.html">RunCCA.default</a></td>
<td>Perform Canonical Correlation Analysis</td></tr>
<tr><td style="width: 25%;"><a href="RunCCA.html">RunCCA.Seurat</a></td>
<td>Perform Canonical Correlation Analysis</td></tr>
<tr><td style="width: 25%;"><a href="RunICA.html">RunICA</a></td>
<td>Run Independent Component Analysis on gene expression</td></tr>
<tr><td style="width: 25%;"><a href="RunICA.html">RunICA.Assay</a></td>
<td>Run Independent Component Analysis on gene expression</td></tr>
<tr><td style="width: 25%;"><a href="RunICA.html">RunICA.default</a></td>
<td>Run Independent Component Analysis on gene expression</td></tr>
<tr><td style="width: 25%;"><a href="RunICA.html">RunICA.Seurat</a></td>
<td>Run Independent Component Analysis on gene expression</td></tr>
<tr><td style="width: 25%;"><a href="RunPCA.html">RunPCA</a></td>
<td>Run Principal Component Analysis</td></tr>
<tr><td style="width: 25%;"><a href="RunPCA.html">RunPCA.Assay</a></td>
<td>Run Principal Component Analysis</td></tr>
<tr><td style="width: 25%;"><a href="RunPCA.html">RunPCA.default</a></td>
<td>Run Principal Component Analysis</td></tr>
<tr><td style="width: 25%;"><a href="RunPCA.html">RunPCA.Seurat</a></td>
<td>Run Principal Component Analysis</td></tr>
<tr><td style="width: 25%;"><a href="RunTSNE.html">RunTSNE</a></td>
<td>Run t-distributed Stochastic Neighbor Embedding</td></tr>
<tr><td style="width: 25%;"><a href="RunTSNE.html">RunTSNE.DimReduc</a></td>
<td>Run t-distributed Stochastic Neighbor Embedding</td></tr>
<tr><td style="width: 25%;"><a href="RunTSNE.html">RunTSNE.matrix</a></td>
<td>Run t-distributed Stochastic Neighbor Embedding</td></tr>
<tr><td style="width: 25%;"><a href="RunTSNE.html">RunTSNE.Seurat</a></td>
<td>Run t-distributed Stochastic Neighbor Embedding</td></tr>
<tr><td style="width: 25%;"><a href="RunUMAP.html">RunUMAP</a></td>
<td>Run UMAP</td></tr>
<tr><td style="width: 25%;"><a href="RunUMAP.html">RunUMAP.default</a></td>
<td>Run UMAP</td></tr>
<tr><td style="width: 25%;"><a href="RunUMAP.html">RunUMAP.Graph</a></td>
<td>Run UMAP</td></tr>
<tr><td style="width: 25%;"><a href="RunUMAP.html">RunUMAP.Seurat</a></td>
<td>Run UMAP</td></tr>
</table>

<h2><a name="S">-- S --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="SampleUMI.html">SampleUMI</a></td>
<td>Sample UMI</td></tr>
<tr><td style="width: 25%;"><a href="ScaleData.html">ScaleData</a></td>
<td>Scale and center the data.</td></tr>
<tr><td style="width: 25%;"><a href="ScaleData.html">ScaleData.Assay</a></td>
<td>Scale and center the data.</td></tr>
<tr><td style="width: 25%;"><a href="ScaleData.html">ScaleData.default</a></td>
<td>Scale and center the data.</td></tr>
<tr><td style="width: 25%;"><a href="ScaleData.html">ScaleData.Seurat</a></td>
<td>Scale and center the data.</td></tr>
<tr><td style="width: 25%;"><a href="ScoreJackStraw.html">ScoreJackStraw</a></td>
<td>Compute Jackstraw scores significance.</td></tr>
<tr><td style="width: 25%;"><a href="ScoreJackStraw.html">ScoreJackStraw.DimReduc</a></td>
<td>Compute Jackstraw scores significance.</td></tr>
<tr><td style="width: 25%;"><a href="ScoreJackStraw.html">ScoreJackStraw.JackStrawData</a></td>
<td>Compute Jackstraw scores significance.</td></tr>
<tr><td style="width: 25%;"><a href="ScoreJackStraw.html">ScoreJackStraw.Seurat</a></td>
<td>Compute Jackstraw scores significance.</td></tr>
<tr><td style="width: 25%;"><a href="SelectIntegrationFeatures.html">SelectIntegrationFeatures</a></td>
<td>Select integration features</td></tr>
<tr><td style="width: 25%;"><a href="SetAssayData.html">SetAssayData</a></td>
<td>Setter for multimodal data</td></tr>
<tr><td style="width: 25%;"><a href="SetAssayData.html">SetAssayData.Assay</a></td>
<td>Setter for multimodal data</td></tr>
<tr><td style="width: 25%;"><a href="SetAssayData.html">SetAssayData.Seurat</a></td>
<td>Setter for multimodal data</td></tr>
<tr><td style="width: 25%;"><a href="Idents.html">SetIdent</a></td>
<td>Get, set, and manipulate an object's identity classes</td></tr>
<tr><td style="width: 25%;"><a href="Idents.html">SetIdent.Seurat</a></td>
<td>Get, set, and manipulate an object's identity classes</td></tr>
<tr><td style="width: 25%;"><a href="SetIntegrationData.html">SetIntegrationData</a></td>
<td>Set integation data</td></tr>
<tr><td style="width: 25%;"><a href="Seurat-class.html">Seurat</a></td>
<td>The Seurat Class</td></tr>
<tr><td style="width: 25%;"><a href="oldseurat-class.html">seurat</a></td>
<td>The Seurat Class</td></tr>
<tr><td style="width: 25%;"><a href="Seurat-class.html">Seurat-class</a></td>
<td>The Seurat Class</td></tr>
<tr><td style="width: 25%;"><a href="oldseurat-class.html">seurat-class</a></td>
<td>The Seurat Class</td></tr>
<tr><td style="width: 25%;"><a href="AddMetaData.html">SeuratAccess</a></td>
<td>Add in metadata associated with either cells or features.</td></tr>
<tr><td style="width: 25%;"><a href="SeuratTheme.html">SeuratAxes</a></td>
<td>Seurat Themes</td></tr>
<tr><td style="width: 25%;"><a href="SeuratCommand-class.html">SeuratCommand</a></td>
<td>The SeuratCommand Class</td></tr>
<tr><td style="width: 25%;"><a href="SeuratCommand-class.html">SeuratCommand-class</a></td>
<td>The SeuratCommand Class</td></tr>
<tr><td style="width: 25%;"><a href="SeuratTheme.html">SeuratTheme</a></td>
<td>Seurat Themes</td></tr>
<tr><td style="width: 25%;"><a href="SeuratTheme.html">SpatialTheme</a></td>
<td>Seurat Themes</td></tr>
<tr><td style="width: 25%;"><a href="DotPlot.html">SplitDotPlotGG</a></td>
<td>Dot plot visualization</td></tr>
<tr><td style="width: 25%;"><a href="SplitObject.html">SplitObject</a></td>
<td>Splits object into a list of subsetted objects.</td></tr>
<tr><td style="width: 25%;"><a href="Idents.html">StashIdent</a></td>
<td>Get, set, and manipulate an object's identity classes</td></tr>
<tr><td style="width: 25%;"><a href="Idents.html">StashIdent.Seurat</a></td>
<td>Get, set, and manipulate an object's identity classes</td></tr>
<tr><td style="width: 25%;"><a href="Stdev.html">Stdev</a></td>
<td>Get the standard deviations for an object</td></tr>
<tr><td style="width: 25%;"><a href="Stdev.html">Stdev.DimReduc</a></td>
<td>Get the standard deviations for an object</td></tr>
<tr><td style="width: 25%;"><a href="Stdev.html">Stdev.Seurat</a></td>
<td>Get the standard deviations for an object</td></tr>
<tr><td style="width: 25%;"><a href="StopCellbrowser.html">StopCellbrowser</a></td>
<td>Stop Cellbrowser web server</td></tr>
<tr><td style="width: 25%;"><a href="subset.Seurat.html">subset</a></td>
<td>Subset a Seurat object</td></tr>
<tr><td style="width: 25%;"><a href="subset.Seurat.html">subset.Seurat</a></td>
<td>Subset a Seurat object</td></tr>
<tr><td style="width: 25%;"><a href="SubsetByBarcodeInflections.html">SubsetByBarcodeInflections</a></td>
<td>Subset a Seurat Object based on the Barcode Distribution Inflection Points</td></tr>
<tr><td style="width: 25%;"><a href="SubsetData.html">SubsetData</a></td>
<td>Return a subset of the Seurat object</td></tr>
<tr><td style="width: 25%;"><a href="SubsetData.html">SubsetData.Assay</a></td>
<td>Return a subset of the Seurat object</td></tr>
<tr><td style="width: 25%;"><a href="SubsetData.html">SubsetData.Seurat</a></td>
<td>Return a subset of the Seurat object</td></tr>
</table>

<h2><a name="T">-- T --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="Tool.html">Tool</a></td>
<td>Get and set additional tool data</td></tr>
<tr><td style="width: 25%;"><a href="Tool.html">Tool.Seurat</a></td>
<td>Get and set additional tool data</td></tr>
<tr><td style="width: 25%;"><a href="Tool.html">Tool&lt;-</a></td>
<td>Get and set additional tool data</td></tr>
<tr><td style="width: 25%;"><a href="Tool.html">Tool&lt;-.Seurat</a></td>
<td>Get and set additional tool data</td></tr>
<tr><td style="width: 25%;"><a href="Tool.html">Tools</a></td>
<td>Get and set additional tool data</td></tr>
<tr><td style="width: 25%;"><a href="TopCells.html">TopCells</a></td>
<td>Find cells with highest scores for a given dimensional reduction technique</td></tr>
<tr><td style="width: 25%;"><a href="TopFeatures.html">TopFeatures</a></td>
<td>Find features with highest scores for a given dimensional reduction technique</td></tr>
<tr><td style="width: 25%;"><a href="TransferData.html">TransferData</a></td>
<td>Transfer Labels</td></tr>
<tr><td style="width: 25%;"><a href="DimPlot.html">TSNEPlot</a></td>
<td>Dimensional reduction plot</td></tr>
</table>

<h2><a name="U">-- U --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="UpdateSeuratObject.html">UpdateSeuratObject</a></td>
<td>Update old Seurat object to accomodate new features</td></tr>
</table>

<h2><a name="V">-- V --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="VariableFeaturePlot.html">VariableFeaturePlot</a></td>
<td>View variable features</td></tr>
<tr><td style="width: 25%;"><a href="VariableFeatures.html">VariableFeatures</a></td>
<td>Get and set variable feature information</td></tr>
<tr><td style="width: 25%;"><a href="VariableFeatures.html">VariableFeatures.Assay</a></td>
<td>Get and set variable feature information</td></tr>
<tr><td style="width: 25%;"><a href="VariableFeatures.html">VariableFeatures.Seurat</a></td>
<td>Get and set variable feature information</td></tr>
<tr><td style="width: 25%;"><a href="VariableFeatures.html">VariableFeatures&lt;-</a></td>
<td>Get and set variable feature information</td></tr>
<tr><td style="width: 25%;"><a href="VariableFeatures.html">VariableFeatures&lt;-.Assay</a></td>
<td>Get and set variable feature information</td></tr>
<tr><td style="width: 25%;"><a href="VariableFeatures.html">VariableFeatures&lt;-.Seurat</a></td>
<td>Get and set variable feature information</td></tr>
<tr><td style="width: 25%;"><a href="VariableFeaturePlot.html">VariableGenePlot</a></td>
<td>View variable features</td></tr>
<tr><td style="width: 25%;"><a href="VizDimLoadings.html">VizDimLoadings</a></td>
<td>Visualize Dimensional Reduction genes</td></tr>
<tr><td style="width: 25%;"><a href="VlnPlot.html">VlnPlot</a></td>
<td>Single cell violin plot</td></tr>
</table>

<h2><a name="W">-- W --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="WhichCells.html">WhichCells</a></td>
<td>Identify cells matching certain criteria</td></tr>
<tr><td style="width: 25%;"><a href="WhichCells.html">WhichCells.Assay</a></td>
<td>Identify cells matching certain criteria</td></tr>
<tr><td style="width: 25%;"><a href="WhichCells.html">WhichCells.Seurat</a></td>
<td>Identify cells matching certain criteria</td></tr>
<tr><td style="width: 25%;"><a href="SeuratTheme.html">WhiteBackground</a></td>
<td>Seurat Themes</td></tr>
</table>

<h2><a name="misc">-- misc --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="subset.Seurat.html">[.Seurat</a></td>
<td>Subset a Seurat object</td></tr>
<tr><td style="width: 25%;"><a href="AddMetaData.html">[[&lt;--method</a></td>
<td>Add in metadata associated with either cells or features.</td></tr>
</table>
</body></html>
